qa/L0_optuna_ensemble_model/check_results.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+# Copyright 2021-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Same issue with the copyright. Could you go through and make sure all copyrights state only 2024 for new files? And double check that the latest copyright text is reflected for new files.